### ISSUE TITLE
Symbol browser show less/scss includes

### ISSUE THREAD

**Issue by babobski** [Komodo/KomodoEdit#3207]
### Short Summary
Would be nice if you could see where there are imports in you're files, in the symbol browser.

![afbeelding](https://user-images.githubusercontent.com/3530262/31046282-483dfce8-a5f6-11e7-97d4-867e0e35b3e5.png)

### Platform Information

Komodo IDE, version 11.0.0, build 90668, platform win32-x86

**Comment by mitchell-as** [comment]
@Naatan the symbol browser is only supposed to show scopes right? (classes, functions, etc.) Not individual entities like variables and imports, right?

**Comment by Naatan** [comment]
@mitchell-as correct, although keep in mind it has a pref where you can have it show "everything". Which I think is the solution to this ticket.

@babobski try hitting the cog button and selecting "Show All Symbols".
